openfe/tests/conftest.py

```diff
@@ -26,7 +26,7 @@ def pytest_configure(config):
 
 def pytest_collection_modifyitems(config, items):
     if (config.getoption("--runslow") or
-        os.environ['OFE_SLOW_TESTS'].lower() == 'true'):
+        os.getenv("OFE_SLOW_TESTS", default="false").lower()):
```

Comment:
don't you still need to evaluate if it's equal to something? (i.e. it doesn't evaluate to a bool right?)